core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/UserApiServiceSpec.scala

```diff
@@ -227,74 +227,84 @@ class UserApiServiceSpec extends ApiServiceSpec {
     }
   }
 
-  it should "handle errors creating a billing project" in withEmptyTestDatabase { dataSource: SlickDataSource =>
```

Comment:
most of the changes below are whitespace. the gist of the test changes are to call pollOperationFutureFail and pollOperationOpFail to test 2 different failure modes in the same test.